mysql-schema/036-webauthn-setting.sql

```diff
@@ -0,0 +1,8 @@
+SET NAMES utf8mb4;
+START TRANSACTION;
+
+INSERT INTO changelog VALUES (36, UNIX_TIMESTAMP());
+
+INSERT INTO server_settings (setting, value) VALUES ('webauthn', '"disabled"');
```

Comment:
Alternatively, maybe use `false` instead of `"disabled"`. Or: `wabauthn.enabled` as a key?

Reply:
I was following the existing pattern used with totp:
https://github.com/curveball/a12n-server/blob/master/mysql-schema/030-totp-setting.sql#L6

Valid options are `disabled`, `optional`, and `required`.

I wouldn't be opposed to adding two fields: `enabled` and `required` where both are booleans.

Reply:
fair, lets keep it!